netbox/utilities/templates/helpers/utilization_graph.html

```diff
@@ -9,13 +9,18 @@
       aria-valuemin="0"
       aria-valuemax="100"
       aria-valuenow="{{ utilization }}"
+      {% if is_full_danger %}
+      class="progress-bar"
+      style="width: {{ utilization }}%; background-color: #800080;"
```

Comment:
Do you have any demo image for this? Including all four colors in light mode and dark mode?

Reply:
Sure, I choose constants color for 100% utilization

Reply:
Darkmode:
![image](https://user-images.githubusercontent.com/43313369/158026742-41f4e2ba-74f0-407c-95d1-6786e78c2607.png)
Lightmode:
![image](https://user-images.githubusercontent.com/43313369/158026763-79b20653-688c-4e7d-ab7f-931d01e05943.png)